plugins/cache.rst

```diff
@@ -33,22 +33,78 @@ control headers from the server as specified in :rfc:`7234`. It needs a
         [$cachePlugin]
     );
 
-By default, responses with no cache control headers are not cached. If you want
-a default cache lifetime if the server specifies no ``max-age``, use::
+Options
+-------
+
+The third parameter to the ``CachePlugin`` constructor takes an array of options. The plugin has 3 options you can
+configure. Their default values and meaning is described by the table below.
+
++---------------------------+---------------+------------------------------------------------------+
+| Name                      | Default value | Description                                          |
++===========================+===============+======================================================+
+| ``default_ttl``           | ``0``         | The default max age of a Response                    |
++---------------------------+---------------+------------------------------------------------------+
+| ``respect_cache_headers`` | ``true``      | Whatever or not we should care about cache headers   |
++---------------------------+---------------+------------------------------------------------------+
+| ``cache_lifetime``        | 30 days       | The minimum time we should store a cache item        |
++---------------------------+---------------+------------------------------------------------------+
+
+.. note::
+
+    A HTTP response may have expired but it is still in cache. If so, headers like ``If-Modified-Since`` and
+    ``If-None-Match`` are added to the HTTP request to allow the server answer with 304 status code. When
+    a 304 response is received we update the CacheItem and save it again for at least ``cache_lifetime``.
+
+Using these options together you may control the how your cached responses behave. By default, responses with no
```

Comment:
"you may control the how your cached responses behave" => you can control how your responses should be cached.